### PR TITLE
StaggerUStride valid numbers

### PR DESCRIPTION
**Pull request by babakpst**
Zero is a valid number for StaggerUSride, but it has not been listed in Common.py.  As a result, any yaml file with SUS=0 will fail. This PR adds zero to the list. 

Additionally, Tensile sorts fork parameters in the output.

### CODE REVIEW

File: Tensile/BenchmarkProblems.py [hunk 1]
```diff
@@ -238,7 +237,7 @@ def benchmarkProblemType(problemTypeConfig, problemSizeGroupConfig, problemSizeG
         print1("# Benchmark Step: {} - {} {:.3f}s".format(groupName, stepName, elapsedTime))
         print1("# Num Sizes: {}".format(benchmarkStep.problemSizes.totalProblemSizes))
         print1("# Fork Parameters:")
-        for k, v in benchmarkStep.forkParams.items():
+        for k, v in sorted(benchmarkStep.forkParams.items()):
```
Comment (nakajee) [inline Tensile/BenchmarkProblems.py:240]: I am not familiar with this part of code.
Please give me some more detail about this change.
What is this for?
How your change works better than before?

File: Tensile/Common.py [hunk 2]
```diff
@@ -783,7 +783,7 @@ def getArchitectureName(gfxName):
     # StaggerUStride will be internally increased so it is an integer multiple of DepthU*BpeAB.
     # (the implementation requires this - the unroll iteration accesses data in steps of
     # DepthU*BPE
-    "StaggerUStride":               [16,32,64,128,256,512,1024],
+    "StaggerUStride":        [0,16,32,64,128,256,512,1024],
```
Comment (nakajee) [inline Tensile/Common.py:786]: Please put some comment here. Like 0 is for StaggerU=0 or something.

File: Tensile/BenchmarkProblems.py (hunk 1 above)
Comment (babakpst) [reply Tensile/BenchmarkProblems.py:240]: This command sorts the fork parameters in the output, so you can find the parameters easier.

Before my changes Tensile output looks like:
# Fork Parameters:
#     StaggerU: [0, 32]
#     StaggerUStride: [128, 256]
#     StaggerUMapping: [1, 2, 3]
#     MatrixInstruction: [[32, 32, 2, 1], [32, 32, 2, 1, 1, 1, 1, 1, 4]]
#     DepthU: [8, 16]

After my changes (ordered alphabetically): 
# Fork Parameters:
#     DepthU: [8, 16]
#     MatrixInstruction: [[32, 32, 2, 1], [32, 32, 2, 1, 1, 1, 1, 1, 4]]
#     StaggerU: [0, 32]
#     StaggerUMapping: [1, 2, 3]
#     StaggerUStride: [128, 256]

File: Tensile/Common.py (hunk 2 above)
Comment (babakpst) [reply Tensile/Common.py:787]: Done, Thanks.

File: Tensile/BenchmarkProblems.py (hunk 1 above)
Comment (nakajee) [reply Tensile/BenchmarkProblems.py:240]: OK. I got it.
That looks good.

**Comment by nakajee** [comment]
Thanks for your update.
I do not have any further comment on the code change.
I found the following fail in the CI test.
Please make sure if this is caused by your change or not.

=============

=========================== short test summary info ============================

FAILED Tensile/Tests/yaml_only/test_config.py::test_config[Tensile/Tests/extended/classic_source/test_sgemm.yaml]

=========== 1 failed, 592 passed, 120 skipped in 15177.61s (4:12:57) ===========

**Review by benjaminulmer — APPROVED** [review]
lgtm. Koji seems happy with the code changes and CI seems happy as well, so I think this is good to go. Please squash-merge when ready

**Pull request merged**